src/central-analytics.rst

```diff
@@ -0,0 +1,22 @@
+
+.. _central-analytics:
+
+Usage Analytics in Central
+==========================
+
+By default, Central does not send any usage analytics.
```

Comment:
This feels like an odd start. I wonder if it'd read better as a note right before the screenshot? 

> tip: By default, Central does not send any usage information to the ODK team. Choosing to share anonymized usage analytics with the Central team is voting for your needs!
>
> You can find the Usage Information settings under ‣ System, then ‣ Usage Information
> 
> screenshot...

Reply:
yeah the tip is perfect. i did want it to be the first sentence.